webapp/src/components/AccessIcon.tsx

```diff
@@ -32,13 +32,16 @@ export default function AccessIcon({ access }: { access: Access }) {
             break
         }
     }
-    return (
+
+    return text ? (
```

Comment:
`text` is always set -- you need to create a new property on the component - like `notext` (default false), and use that

Reply:
You mean `return notext`?